client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/components/credentials-form.tsx

```diff
@@ -43,6 +47,10 @@ export const CredentialsForm: FC< CredentialsFormProps > = ( { onSubmit, onSkip
 
 	const getContinueButtonText = () => {
 		if ( isBusy && ! canBypassVerification ) {
+			const hasScanningTranslation = locale.startsWith( 'en' ) || hasTranslation( 'Scanning site' );
```

Comment:
@valter, check out [this hook](https://github.com/Automattic/wp-calypso/blob/f85d691931d17f048281b631ff62b999e72bb57b/packages/i18n-utils/src/locale-context.tsx#L136-L143). I think it will lead to simpler code. Here's a [sample usage PR](https://github.com/Automattic/wp-calypso/pull/96000/files).